importer/templates/match.html

```
@@ -116,9 +118,9 @@
                 No results found for this search...
             </div>
             <div style="padding-left: 10px;">
+                <button class="button" onclick="closeSearchPanel()">Cancel</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)